### PR TITLE
ci: Add github workflow for nitro node

### PR DESCRIPTION
**Pull request by InNoobWeTrust**
This is a successor of janhq/nitro#367
- Fix bug not being able to spawn nitro process on Windows
- Add CI workflows for nitro-node

### CODE REVIEW

File: nitro-node/src/nitro.ts [hunk 1]
```diff
@@ -214,23 +217,17 @@ async function runNitroAndLoadModel(
      * The tested threshold is 500ms
      **/
     if (process.platform === "win32") {
-      return await new Promise((resolve) => setTimeout(() => resolve({}), 500));
-    }
-    const spawnResult = await spawnNitroProcess(runMode);
-    if (spawnResult.error) {
-      return spawnResult;
+      await new Promise((resolve) => setTimeout(resolve, 500));
```
Comment (InNoobWeTrust) [inline nitro-node/src/nitro.ts:220]: Hello @louis-jan, can you help review this PR? There is a serious bug at this line that I overlooked. The extra `return` makes the library unusable on Windows as the code will return early before spawning nitro process (my bad). 😫

Comment (louis-jan) [reply nitro-node/src/nitro.ts:220]: Sure @InNoobWeTrust. Thank you

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@ name: CI
 
 on:
   schedule:
-    - cron: "0 20 * * *" # At 8 PM UTC, which is 3 AM UTC+7
```
Comment (louis-jan) [inline .github/workflows/build.yml:5]: It's not relevant to the PR right?

Comment (InNoobWeTrust) [reply .github/workflows/build.yml]: yes, reverted the unrelated change. 😉

**Review by louis-jan — APPROVED** [review]
LGTM

**Comment by louis-jan** [comment]
We are working on Cortex NPM and CLI so we could close this now.

**Comment by InNoobWeTrust** [comment]
> We are working on Cortex NPM and CLI so we could close this now.

Great, I'm waiting for the official NPM package then. 😆

**Pull request closed without merging**